persistence/relational-jdbc/src/main/resources/h2/schema-v2.sql

```diff
@@ -63,6 +63,9 @@ CREATE INDEX IF NOT EXISTS idx_locations ON entities(realm_id, catalog_id, locat
 
 -- TODO: create indexes based on all query pattern.
 CREATE INDEX IF NOT EXISTS idx_entities ON entities (realm_id, catalog_id, id);
+CREATE INDEX IF NOT EXISTS idx_entities_lookup
+    ON entities (realm_id, catalog_id, parent_id, type_code, sub_type_code)
+    INCLUDE (id, name);
```

Comment:
according to https://h2database.com/html/commands.html#create_index `INCLUDE` can only be used on a unique index in H2

did we confirm this works?